apps/dashboard/test/system/remote_files_test.rb

```diff
@@ -4,6 +4,11 @@
 class RemoteFilesTest < ApplicationSystemTestCase
   MAX_WAIT = 120
 
+  def setup
+    # we want to clear the console logs from any previous test.
+    Capybara.current_session.quit
+  end
```

Comment:
This is not really a part of this PR, but following #3607 we need to apply this here as well to clear any console logs before this test file checks them.